app/components/mandatarissen/mandataris-edit-prompt.js

```diff
@@ -63,6 +73,13 @@ export default class MandatenbeheerMandatarisEditPromptComponent extends Compone
     }
   }
 
+  @action
+  async onBekrachtig() {
+    this.editMode = null;
+    this.args.mandataris.isDraft = false;
+    this.args.mandataris.save();
```

Comment:
Not sure if sending to the backend is necessary

Reply:
why wouldn't it be? if not your changes would be gone?